cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -86,24 +103,34 @@ function printMembers () {
 // Output the hacker data in JSON format to sortedHackers.json
 function writeDataToFile () {
   const hackerTables = JSON.stringify(hackerList)
-  const pathToWrite = path.resolve(__dirname, 'data', 'json_outputs', 'sortedHackers.json')
+  const pathToWrite = path.resolve(
+    __dirname,
+    'data',
+    'json_outputs',
+    'sortedHackers.json'
+  )
   fs.writeFileSync(pathToWrite, hackerTables)
 }
 
 // Assigns two cabins to each Hacker.
 // assignedCabin = the cabin a Hacker is best suited to
 // secondAssignedCabin = their next best cabin option
-function hackerSortingAlgo () {
+async function hackerSortingAlgo () {
   // variable values first declared globally within the file and initialized on runtime
-  hackerList = loadCSV('hackerData.csv', true)
-  answerList = loadCSV('answer.csv', true)
-  cabinList = loadCSV('cabinTypes.csv', false)[0]
+  // hackerList = loadCSV('hackerData.csv', true)
+  hackerList = await grabFromDatabase()
+  answerList = loadCSV('answer.csv', true, '|')
```

Comment:
csv is comma separated and when we get form data that is how it will be. it is an unnecessary extra step to convert to using `|`. please convert this and sample data and verify your code still works!